components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -3188,6 +3195,12 @@ public void removeSubscription(SubscribedAPI subscription) throws APIManagementE
                         + ") removed from app " + appName;
                 log.debug(logMessage);
             }
+
+            SubscriptionEvent subscriptionEvent = new SubscriptionEvent(UUID.randomUUID().toString(),
```

Comment:
These Events need to send after subscription approved

Reply:
resolved

Reply:
still not fixed